packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/index.md

```diff
@@ -95,8 +50,7 @@ At this point, you can move to the next step: [Creating your app](#create-app).
 1. Enter an **App integration name**.
 1. Enter the callback routes.
 
-   * For the **Sign-in redirect URIs**, enter the [full redirect URI](#define-a-callback-route) for your mobile app (for example, `com.okta.example:/callback`).
-   * For the **Sign-out redirect URIs**, enter the [full redirect URI](#define-a-callback-route) for your mobile app (for example, `com.okta.example:/logout`).
+    <StackSnippet snippet="redirectvalues" />
```

Comment:
not sure how this change relates to removing CLI references; can someone confirm that it renders correctly? I'm not set up to preview rendering on the dev docs at the moment

Reply:
> Netlify Preview URL for the changes: https://preview-5449--reverent-murdock-829d24.netlify.app